app-deployment-manager/VERSION

```diff
@@ -1 +1 @@
-2.3.45-dev
+2.3.45
```

Comment:
the version on main is 2.4.3 not sure how it ended up this version

Reply:
this is going on `release-3.0` branch, so I think it's right that it's 2.**3**.x here